### PR TITLE
adding script for installing sysdig agent on iks cluster

### PR DESCRIPTION
**Pull request by arsarana**

### CODE REVIEW

File: agent_deploy/IBMCloud-Kubernetes-Service/install-agent-k8s.sh [hunk 1]
```diff
@@ -0,0 +1,337 @@
...
+}
+
+function unsupported {
+	echo "Unsupported operating system. Try using the the manual installation instructions"
+	exit 1	
+}
+
+function help {
+	echo "Usage: $(basename ${0}) -a | --access_key <value> [-t | --tags <value>] [-c | --collector <value>] \ "
+    echo "                [-cp | --collector_port <value>] [-s | --secure <value>] [-cc | --check_certificate] \ "
```
Comment (nestorsalceda) [inline agent_deploy/IBMCloud-Kubernetes-Service/install-agent-k8s.sh]: Hey, I think this is a problem with indentation, could you please fix it??

Thanks!

Comment (arsidada) [reply agent_deploy/IBMCloud-Kubernetes-Service/install-agent-k8s.sh]: Fixed!

**Comment by arsidada** [comment]
Thanks for the review. Fixed the indentation issues. Ready to for re-review now

File: agent_deploy/IBMCloud-Kubernetes-Service/install-agent-k8s.sh (hunk 1 above)
Comment (arsarana) [reply agent_deploy/IBMCloud-Kubernetes-Service/install-agent-k8s.sh]: Fixed!

**Comment by arsarana** [comment]
Fixed the indentation issues - ready for re-review!

**Pull request merged**